features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -106,7 +106,7 @@ CREATE TABLE IDN_OAUTH2_TOKEN_BINDING (
             TOKEN_BINDING_REF VARCHAR (32) NOT NULL,
             TOKEN_BINDING_VALUE VARCHAR (1024) NOT NULL,
             TENANT_ID INTEGER DEFAULT -1,
-            UNIQUE (TOKEN_ID,TOKEN_BINDING_TYPE,TOKEN_BINDING_VALUE),
+            UNIQUE (TOKEN_ID,TOKEN_BINDING_TYPE,TOKEN_BINDING_VALUE) INDEX IN TS32K,
```

Comment:
Should this be reflected in https://github.com/wso2-extensions/identity-migration-resources/pull/175/files#diff-e5b41276c6dd36d7a85c3b919b73fd7d7e3cb3962bc6b86b411d6e41a3bb0eb8R1 as well?

Reply:
yes. It should be added to migration client as well. I'm working on that.